### PR TITLE
Resolves #2348, corrects issue with missing ARIA helper

### PR DESCRIPTION
**Pull request by brian-learningpool**
This was due to the supported framework version being out of step.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-narrative",
-  "version": "3.0.2",
-  "framework": ">=3.2",
+  "version": "3.0.3",
+  "framework": "^3.2",
```
Comment (tomgreenfield) [inline bower.json]: I'd suggest against reintroducing the caret `^` – it generally causes more issues than it fixes.

For example, when importing older courses into the authoring tool, the old `^2.0.0` dependencies were causing headaches, even though the plugins were compatible with the higher framework version.

There's nothing in this version of Narrative that _doesn't_ cause it to work with framework 4.

Comment (brian-learningpool) [reply bower.json]: So with the v4.0.0 version re-tagged is this PR even required at all?

Comment (tomgreenfield) [reply bower.json]: Not unless I'm missing something...@moloko? @oliverfoster?

Comment (brian-learningpool) [reply bower.json]: 👍 I'll close it then.

Comment (moloko) [reply bower.json]: yeah sorry that could be my bad, I saw a reference to `a11y_aria_label` in narrative.hbs in 3.0.2 and thought that was a problem - but it's actually `a11y_aria_image` that's the problematic helper (in that it doesn't exist in FW v3.x)

**Pull request closed without merging**